deb/build/debian/jenkins.init

```diff
@@ -43,6 +46,26 @@ if [ "$RUN_STANDALONE" = "false" ]; then
     exit 1
 fi
 
+# Make sure there exists a java executable, it may not be allways the case
+if [ -z "$JAVA" ]; then
```

Comment:
@samrocketman I believe @oleg-nenashev is referring to use `JAVA_HOME` as location of the java executable if present